hail/src/main/scala/is/hail/expr/ir/analyses/CompilationUnitSize.scala

```diff
@@ -0,0 +1,21 @@
+package is.hail.expr.ir.analyses
+
+import is.hail.expr.ir._
+
+object CompilationUnitSize {
```

Comment:
This seems like it's not used anywhere?

Reply:
oh, good catch! This was used in an earlier version.

Reply:
Ah ok. Feel free to dismiss my review after deleting it.